src/main/java/jenkins/scm/api/mixin/ChangeRequestSCMHead.java

```diff
@@ -47,6 +47,30 @@
     @NonNull
     String getId();
 
+    /**
+     * Returns the {@link ChangeRequestCheckoutStrategy} of this {@link ChangeRequestSCMHead}.
+     *
+     * @return the {@link ChangeRequestCheckoutStrategy}.
+     */
+    default ChangeRequestCheckoutStrategy getCheckoutStrategy() {
+        return null;
+    }
+
+    /**
+     * Returns the name of the actual head on the source control system which may or may not be different from
+     * {@link #getName()}. For example in GitHub or Bitbucket this method would return the name of the origin branch
+     * whereas {@link #getName()} would return something like {@code PR-24}. It is perfectly acceptable for a SCM
+     * implementation to return the same value as {@link #getName()} where the SCM implementation does not have a
+     * separate concept of origin name.
+     *
+     * @return the name this {@link ChangeRequestSCMHead} would have if the {@link SCMSource} were configured
+     * against the {@link #getOrigin()} directly and the change request were be discoverable as a regular
+     * {@link SCMHead} or {@link #getName()} if such a concept is not possible in the backing source control system.
```

Comment:
```suggestion
     * {@link SCMHead} or {@link #getName()} if such a concept is not possible in the backing source control system.
     *
     * @since TODO
```